controller/xweb/contextutil.go

```diff
@@ -18,9 +18,9 @@ package xweb
 
 import "context"
 
-// WebHandlerFromContext us a utility function to retrieve a WebHandler reference, that the demux http.Handler
+// WebHandlerFromRequestContext us a utility function to retrieve a WebHandler reference, that the demux http.Handler
```

Comment:
http.Request xweb context merge. Instead of multiple just 1 for core info and 1 for handlers.